backend-api/app/routes/consumers.py

```diff
@@ -6,24 +6,32 @@
 
 router = APIRouter()
 
-consumer_dict = {"ethereum_address": "somewhareOnTheBlockchain"}
+consumer_dict = {"eth_address": "somewhareOnTheBlockchain"}
 
 
 @router.post("/", status_code=status.HTTP_201_CREATED)
 async def create_consumer(consumer: ConsumerCreate):
+    # Create database instance, retrieve ID
+
     return ConsumerRead(id=random.randint(1, 1000), **consumer_dict)
 
 
-@router.get("/me")
-async def read_consumer():
+@router.get("/{eth_address}}", status_code=status.HTTP_200_OK)
+async def read_consumer(eth_address: str):
+    # Query database instance
+
     return ConsumerRead(id=random.randint(1, 1000), **consumer_dict)
 
 
-@router.patch("/me")
-async def update_consumer(consumer: ConsumerUpdate):
-    return ConsumerRead(id=random.randint(1, 1000), **consumer_dict)
+@router.patch("/{eth_address}", status_code=status.HTTP_204_NO_CONTENT)
+async def update_consumer(eth_address: str, consumer: ConsumerUpdate):
+    # Update database instance
+
+    return None
+
 
+@router.delete("/{eth_address}", status_code=status.HTTP_204_NO_CONTENT)
+async def delete_consumer(eth_address: str):
+    # Delete database instance
 
-@router.delete("/me", status_code=status.HTTP_204_NO_CONTENT)
-async def delete_consumer():
-    return
+    return None
```

Comment:
Returning None isn't the same as no content, since None is something and can be converted to JSON, while nothing is nothing and isn't a JSON object.